tests/test_spaces.py

```diff
@@ -56,6 +57,21 @@ def step(self, action):
         return self.observation_space.sample(), 0.0, False, False, {}
 
 
+class DummyEnv(gym.Env):
```

Comment:
I propose that we replace the other `DummyEnvs` in `test_spaces.py` and `test_buffers.py` (and maybe more test files) with this one. This would make the tests more modular and provide more overview. If you agree, I would open a new issue and create a PR in the upcoming weeks.

Reply:
thanks for the proposal, it might actually be helpful to include directly in the library or in gymnasium (I still need to think more about it).

Reply:
Done in https://github.com/DLR-RM/stable-baselines3/pull/1572/commits/985805bf5f32b171ba199d3ccb9960889f2f4483 =)